helm/botkube/values.yaml

```diff
@@ -564,12 +564,15 @@ sources:
             type: Static
             static:
               values: [*argocd-group]
+      # -- Config contains configuration for ArgoCD source plugin.
+      # This section lists only basic options, and uses default triggers and templates which are based on ArgoCD Notification Catalog ones.
```

Comment:
What about giving the catalog link here? I meant this URL 👉 https://argocd-notifications.readthedocs.io/en/latest/catalog/